### ISSUE TITLE
🐛 [Bug Report] - Tauren - War Stomp triggers the GCD

### ISSUE THREAD

**Issue by schanzee** [Yatzii93/TBC5MAN-Bug-Tracker#120]
# Decription: #
you cant warstomp someone and cyclone him before the stun falls off for example
# How it works: #
war stomp triggers the gcd
# How it should work: #
you should be able to cast immediately once warstomps casting time is finished
# Source - You need to provide a source as evidence: #
https://wowwiki.fandom.com/wiki/War_Stomp?oldid=1446536
https://www.youtube.com/watch?v=Gqg1vEWEILU at 4:28 and 8:18
# Additional Information: #

**Comment by Yatzii93** [comment]
Tested and not the case. Please show clear evidence that the raciel spell Warstomp is causes the GCD to trigger because as far as I can see it does not and is working as intended. Closing for now will open if sufficient evidence is provided.

**Comment by Yatzii93** [comment]
Aditionally please stop re-creating issues there is already a ticket for, instead of just copying and pasting why not provide additional information to back up your claim in the last issue you raised. https://github.com/Yatzii93/TBC5MAN-Bug-Tracker/issues/116. Please use the bugtracker properly next time. Either update this issue or the old one with new information or don't do anything. Thanks.

**Comment by schanzee** [comment]
https://youtu.be/_ddZYOGV_CY
after warstomp theres half a second where you cannot cast very noticable and you cannot safely get a 1.5 second cast to finish before the 2 second stun expires.

you tag issues with lacking information or proof soon after opening and immediiately close them at the same time.
i've added info to other reports after and they're just not been updated at all for weeks or months. if you could give more feedback of any sort that would really help. 
if you dont users never know if its still not enough sources if its low priority and you'll maybe take a look when you have time or you just missed it for some reason and since the issue is closed it will never be looked at again.
i'll try to put more effort into future bug reports some things just seem so obvious they dont require whole essay to explain just someone to point them out.

**Comment by Yatzii93** [comment]
Im not sure what you think GCD is but what you just showed proves that it does infact not trigger the GCD. What you are seeing is the time it takes for the animation to go off. If you're stomping the ground you aren't going to be casting. This is normal behaviour.

**Comment by schanzee** [comment]
https://www.youtube.com/watch?v=Gqg1vEWEILU at 8:18 you can see the druid casting first warstomp then a cyclone back to back seemless. the 1.5 cast is through well before the stun falls off. thats not possible on this server.
maybe the animation starts too late here or it lasts for too long i dont know but can you see the difference?

**Comment by schanzee** [comment]
if i get melee hit pushback when casting the warstomp theres no delay with the following cast. just like there was gcd on it but its gone by the time the cast is finished.
https://www.youtube.com/watch?v=Q1RW3iHgs1M
